source/guides/account/scams-and-security.rst

```diff
@@ -0,0 +1,112 @@
+
+Aggregate transactions can include a variety of elements such as multiple signers, messages, and transfers. New Symbol users may not be familiar with the complex nature of Aggregate Bonded transactions, and must take great care to fully understand the full scope of any transactions they are asked to sign. For more information see :doc:`Aggregate Transaction <../../concepts/aggregate-transaction>`
+
+What is sweeping?
+=================
+
+Sweeping involves a malicious script that monitors certain types of transactions, then initiates a malicious Aggregate Bonded transaction disguised to appear related to the targeted transaction (i.e. by including a relevant message), but which also include a transfer of XYM or other mosaics to the bad actor's wallet. Signing these transactions can completely empty a user's wallet, so please make sure you carefully review before signing any transactions that you did not initiate.
+
+It may not be immediately apparent that you've been hacked. The sweeper's goal is that you assume that the transaction you signed was part of a process you were completing, and you may also assume that your balance has been reduced by a stuck transaction, pending activation of delegated harvesting, or that your balance was transferred to a delegated harvesting address linked to your account.
+
+What could a sweeping attack look like?
+========================================
+
+- You begin Delegated Harvesting activation by propagating the 'Link all keys' transaction to the network.
+- The bad actor's script detects this transaction and sends you an Aggregate Bonded transaction. This transaction might include a message related to harvesting such as "Delegated-№234567" as well as a transfer of XYM to an address controlled by the bad actor. 
+- Your wallet sees the Aggregate Bonded transaction included in a subsequent block and prompts you with the option to review and sign it.
+- You choose to sign the transaction, and your XYM balance is reduced by the transaction amount (typically all or nearly all of your XYM).
+
+Every Symbol wallet is different; some may not permit users to sign aggregate transactions, others may hide bonded aggregate transactions originating from addresses outside a predefined whietlist (i.e. your contacts), and others may prompt for users for their signature without revealing the full contents of what you are signing. Reviewing aggregate transactions using the `Symbol Explorer <https://testnet.symbol.fyi/nodes>`__ before signing them can provide additional security and peace of mind. Below is an example of a  malicous aggregate transaction that contains a message to a trusted address in the first transaction and an asset transfer to the scammer's address in the second transaction.
+
+   .. figure:: ../../resources/images/screenshots/Aggregate-transaction.png
+      :align: center
+      :width: 75%
+      :class: with-shadow
+      :target: /_images/Aggregate-transaction.png
```

Comment:
```suggestion
.. figure:: ../../resources/images/screenshots/Aggregate-transaction.png
   :align: center
   :width: 75%
   :class: with-shadow
   :target: /_images/Aggregate-transaction.png
```

The extra indentation adds a white border to the left of the paragraph as if this was a quote. We don't want that.